.gitignore

```
@@ -130,4 +130,4 @@ dmypy.json
 
 # Code Editor
 .vscode
-.idea
\ No newline at end of file
+.idea.DS_Store
```

```suggestion
.idea
.DS_Store
```
The newline is required in this file.